packages/ui/app/src/App.tsx

```diff
@@ -38,7 +39,9 @@ export const App: React.FC<App.Props> = ({ docs, resolvedUrlPath }) => {
                     <Context>{children}</Context>
                 ),
                 <DocsContextProvider docsDefinition={docs.definition} resolvedUrlPath={resolvedUrlPath}>
-                    <Docs />
+                    <SearchContextProvider>
+                        <Docs />
+                    </SearchContextProvider>
```

Comment:
dope thanks. can also put in [contexts.ts](https://github.com/fern-api/fern/blob/main/packages/ui/app/src/contexts.tsx) since it doesn't need state

```tsx
// contexts.ts
export const CONTEXTS = [
    ({ children }: PropsWithChildren): JSX.Element => <ThemeProvider theme="dark">{children}</ThemeProvider>,
    ({ children }: PropsWithChildren): JSX.Element => <SearchContextProvider>{children}</SearchContextProvider>,
];
```

nbd but a nice pattern as contexts lists grows